ClippingV4/Clipping API.postman_collection.json

```
@@ -0,0 +1,4311 @@
+{
+	"info": {
+		"_postman_id": "66825806-bfe7-4eae-aa59-717062f7ead1",
+		"name": "Clipping API",
+		"description": "The API provides CRUD operations to clipping",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "13338628"
+	},
+	"item": [
+		{
+			"name": "Api Spec",
+			"item": [
+				{
+					"name": "Get clip",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Response should be 200\", () => {",
+									"    pm.response.to.have.status(200);",
+									"    pm.response.to.be.not.empty;",
+									"    pm.response.to.be.json;",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "apikey",
+							"apikey": [
+								{
+									"key": "value",
+									"value": "Apikey {{apiKey}}",
+									"type": "string"
+								},
+								{
+									"key": "key",
+									"value": "Authorization",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [
+							{
+								"key": "Accept",
+								"value": "application/vnd.api+json"
+							},
+							{
+								"key": "x-mock-response-code",
+								"value": "200",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "{{baseUrl}}/clips/:clip_id",
```

I think `baseUrl` in environment variables is something like `https://testing.booxmedia.xyz`
```suggestion
							"raw": "{{baseUrl}}/api/clipping/v4/clips/:clip_id",
```

---

fixed